plottable.d.ts

```diff
@@ -3410,6 +3410,19 @@ declare module Plottable {
             interpolator(interpolator: "cardinal-open"): Line<X>;
             interpolator(interpolator: "cardinal-closed"): Line<X>;
             interpolator(interpolator: "monotone"): Line<X>;
+            /**
+             * Gets the croppedRendering preformance option state.
+             *
+             * When croppedRendering option is enabled, lines that will not be visible in the viewport will not be
+             * drawn anymore (will not have corresponding SVG nodes). If only part of the data is in the viewport,
```

Comment:
"Thus if only part of the data is visible, then this option will boost rendering."?